generators/server/templates/src/main/java/package/config/StaticResourcesWebConfiguration.java.ejs

```diff
@@ -33,8 +33,8 @@ import java.util.concurrent.TimeUnit;
 @Profile({JHipsterConstants.SPRING_PROFILE_PRODUCTION})
 public class StaticResourcesWebConfiguration implements WebMvcConfigurer {
 
-    protected static final String[] RESOURCE_LOCATIONS = new String[]{"classpath:/static/app/", "classpath:/static/content/", "classpath:/static/i18n/"};
-    protected static final String[] RESOURCE_PATHS = new String[]{"/app/*", "/content/*", "/i18n/*"};
+    protected static final String[] RESOURCE_LOCATIONS = new String[]{"classpath:/static/", "classpath:/static/content/", "classpath:/static/i18n/"};
+    protected static final String[] RESOURCE_PATHS = new String[]{"/*.js", "/*.css", "/*.svg", "/*.png", "*.ico", "/content/*", "/i18n/*"};
```

Comment:
Would it be possible to include subfolders of `/content/` and their content:

```suggestion
    protected static final String[] RESOURCE_PATHS = new String[]{"/*.js", "/*.css", "/*.svg", "/*.png", "*.ico", "/content/**", "/i18n/*"};
```

Reply:
That should be possible.